web/modules/custom/collection_page/config/install/core.entity_view_display.node.collection_page.teaser.yml

```diff
@@ -0,0 +1,27 @@
+langcode: en
```

Comment:
We don't have a use for this "teaser" display mode. This is enabled because we copied it from the standard profile. Let's remove it now, otherwise we will carry this unused display mode along for all eternity :)

You can remove it by deleting the `core.entity_view_mode.node.teaser` config file from the install profile.